packages/discord.js/src/managers/GuildMemberManager.js

```diff
@@ -441,6 +441,32 @@ class GuildMemberManager extends CachedManager {
     return this.guild.bans.remove(user, reason);
   }
 
+  /**
+   * Adds a role to a member.
+   * @param {GuildMemberResolvable} user The member to add a role to
+   * @param {RoleResolvable} role Options for the role adding
+   * @param {string} [reason]
+   * @returns {Promise<void>}
+   */
+  async addRole(user, role, reason) {
```

Comment:
Why is the parameter named as user if it's supposed to take a member?

Reply:
Well it's also that way for https://discord.js.org/#/docs/discord.js/main/class/GuildMemberManager?scrollTo=kick

Reply:
It uses `UserResolvable`. (But should use `GuildMemberResolvable`. It looks like this is a bug.)
![image](https://user-images.githubusercontent.com/62868468/182036189-09290cc5-8c1f-4c8d-9877-36e4e9be27ce.png)

Reply:
It looks like these methods should also use `UserResolvable`, because not all members are cached and User object won't be resolved.

Reply:
You don't need the member to be cached to resolveId